SU-HKKU/cardinal_service_profile
# CSD-201 Style Bug Fixes

# READY FOR REVIEW

# Summary
- Style Bug Fixes 

# Needed By (Date)
- 8/17

# Urgency
- medium

# Steps to Test

1. Pull this branch
2. Clear cache
3. Check for the following (to be updated shortly):

- Location, When, Open To should now always span 9 columns even when there isn't an image.
- The selected item slug/pull within the filters should be light blue with white text.
- Sort By options should now be styled (additional spacing between text and icons, adjusted link hover and active styles)
- Resource card view should now display 3 in a row and be left aligned. This style will not affect individual Related Resource page views
- Card style should not apply on the secondary nav.


# Associated Issues and/or People
- Primary Story: [CSD-201](https://stanfordits.atlassian.net/browse/CSD-201)
- No Img span 9: [CSD-322](https://stanfordits.atlassian.net/browse/CSD-322)
- Visual style of selected item for Filters: [CSD-297](https://stanfordits.atlassian.net/browse/CSD-297)
- Style Sort By Options: [CSD-313](https://stanfordits.atlassian.net/browse/CSD-313)
- Update Resource Cards: [CSD-314](https://stanfordits.atlassian.net/browse/CSD-314)
- Card styles: [CSD-333](https://stanfordits.atlassian.net/browse/CSD-333)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
